packages/flutter/lib/src/rendering/sliver.dart

```diff
@@ -196,7 +196,22 @@ class SliverConstraints extends Constraints {
   /// the earliest visible part of this sliver in the [AxisDirection].
   ///
   /// For example, if [AxisDirection] is [AxisDirection.down], then this is the
-  /// scroll offset at the top of the visible portion of the sliver.
```

Comment:
i think we should leave this deleted text, since it is the precise definition of the field. (adding more explanatory text is fine)

Reply:
Added the original back. I wanted to make the interval endpoints clearer. The original phrasing was a bit recursive.